cmd/manager/main.go

```diff
@@ -20,6 +20,7 @@ import (
 	"flag"
 	"fmt"
 	"os"
+	env "runtime"
```

Comment:
why env ?

Reply:
runtime was already used by another import. And this helps us log the OS/ARCH & Go version and path which are env on which the image was built.

Reply:
```
time="2021-03-11T10:03:46Z" level=info msg="Go Version: go1.15.5"
time="2021-03-11T10:03:46Z" level=info msg="Go OS/Arch: linux/amd64"
time="2021-03-11T10:03:46Z" level=info msg="Version of jiva-operator: scaleup-dev"
```

Reply:
ok if runtime alias was used then its fine.